content/en/metrics/custom_metrics.md

```diff
@@ -2,13 +2,13 @@
 title: Custom Metrics
 kind: documentation
 aliases:
-  - /metrics/
```

Comment:
this one got removed to make sure we're not aliasing/overwriting the existing metrics index with this page, in case that's not super clear